library/Notifications/Widget/Timeline.php

```diff
@@ -306,7 +307,11 @@ protected function assembleSidebarEntry(Rotation $rotation): BaseHtmlElement
         $entry = new HtmlElement('div', Attributes::create(['class' => 'rotation-name']));
 
         $form = new MoveRotationForm();
-        $form->setAction(Links::moveRotation()->getAbsoluteUrl());
+        $form->setAction(
+            Links::moveRotation()
+                ->with(['display_timezone' => ScheduleDateTimeFactory::getDisplayTimezone()->getName()])
+                ->getAbsoluteUrl()
```

Comment:
The `rotationAdd` and `rotationSettings` handle this themselves, so I think `moveRotation` should do the same.